integration-tests/features/component_analysis_v2_batchcall.feature

```diff
@@ -0,0 +1,87 @@
+Feature: Component analysis V2 Batch Call API
+    @cav2
+    Scenario: Check the component analysis V2 REST API Batch Call for NPM ecosystem
+        Given System is running
+        Given Three scale preview service is running
+
+        When I acquire the use_key for 3scale
+        Then I should get the proper user_key
+        When I start CA batch call for test_npm.json with user_key
+        Then I should get 200 status code
+         And I should receive a valid JSON response
+        Then I should be able to validate all the feilds or vulnerablities in the result
+        And I should find package sequence 3.0.0 has no recommendation
+        And I should find package ejs 1.0.0 has 3.1.3 recommended version
+        Then I should find snyk id SNYK-JS-EJS-10218 and 8.1 for package ejs and version 1.0.0
+        Then I should find snyk id SNYK-JS-ANGULAR-471882 for package angular and version 1.0.0 as private
+    
+     @cav2
+    Scenario: Check the component analysis V2 REST API Batch Call for Maven ecosystem
```

Comment:
Can you add a test for free tier and registered user

Reply:
> Check for CA no Vulnerability scenario,
> Invalid request,

@deepak1725  Done added No Vulnerability scenario and invalid requests

Reply:
@prashbnair need to add a uuid in vault and modify the cent os ci jobs to check for registered user , for now i am checking only for free tier, will add registered user test cases after adding the User management tests